### PR TITLE
Enable MultiAZ support for Openstack with a feature flag

### LINKED ISSUE #4909: Multi AZ support for OpenStack
**User Story**

OpenStack datacenters contain currently a fixed availability zone in the datacenters.yaml / CRD. To support HA clusters over multiple AZs, the zone should be selectable when creating a machine deployment.

This is already supported by the machine-controller, but missing in the Kubermatic API / UI. Editing a machine-deployment from the UI currently sets it to the AZ, configured for the datacenter.

**Acceptance criteria**

Availability zone is selectable during machine deployment creation from the UI

### PR DESCRIPTION
**Pull request by moelsayed**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #4909

**Special notes for your reviewer**:
This was _not_ functionally tested because we don't currently have access to a multiAZ openstack environment. A feature flag was added to enable this later.
**Documentation**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: api/pkg/crd/kubermatic/v1/cluster.go [hunk 1]
```diff
@@ -117,6 +117,9 @@ const (
 	// ClusterFeatureRancherIntegration enables the rancher server integration feature.
 	// It will deploy a Rancher Server Managegment plane on the seed cluster and import the user cluster into it.
 	ClusterFeatureRancherIntegration = "rancherIntegration"
+
+	// ClusterFeatureOpenstackMultiAZ enables OpenStack multiple availability zones zupport
```
Comment (moadqassem) [inline api/pkg/crd/kubermatic/v1/cluster.go]: typo: zupport/support

**Review by moadqassem — CHANGES_REQUESTED** [review]

**Comment by moadqassem** [comment]
@moelsayed Is this actually supported in the UI or how the user gets to choose this option?

**Comment by moelsayed** [comment]
@moadqassem Actually no. Per the open issue, it needs API then UI support. I think I will have to open a UI ticket ? Or assign it to UI team ?

**Comment by moadqassem** [comment]
cc @zreigz PTAL

**Pull request merged**